index.js

```diff
@@ -310,6 +310,20 @@ module.exports = {
       return pull(ssb.box2.listGroupIds({ live: !!opts.live }), paraMap(get, 4))
     }
 
+    function getRootIdOfMsgAuthor(groupRootMsgId, cb) {
```

Comment:
```suggestion
    function getRootFeedIdFromMsg(groupRootMsgId, cb) {
```

I didn't quite understand your function name. I think it should be something like this.
I think this surfaces that we have a need for a new name. Author is kinda feed-specific in current ssb.
I wonder if it's `deviceId` - the feedId of the root meta-feed for a device. God.
`rootAuthor`

`getRootAuthorByMsgId`

Reply:
oh yeah that's a better function name